README.md

```diff
@@ -293,6 +293,7 @@ First, ensure your k8s cluster can pull edge controller image. If the
 image not exist. We can make one, and push to your registry.
 
 ```bash
+cd $GOPATH/src/github.com/kubeedge/kubeedge
 make cloudimage
 ```
```

Comment:
After building the cloud image , we should specify to change the directory to : 
$GOPATH/src/github.com/kubeedge/kubeedge/build/cloud

and then execute this command : 
../tools/certgen.sh buildSecret | tee ./06-secret.yaml

Reply:
+1